tests/e2e/raw_block_volume.go

```diff
@@ -152,20 +164,17 @@ var _ = ginkgo.Describe("raw block volume support", func() {
 		"Statefulset testing with raw block volume and default podManagementPolicy", func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
-		ginkgo.By("Set Resource quota for GC")
+
+		storageClassName = defaultNginxStorageClassName
 		if vanillaCluster {
-			storageClassName = defaultNginxStorageClassName
-			scParameters = nil
+			ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
 		} else if guestCluster {
+			ginkgo.By("CNS_TEST: Running for GC setup")
 			storageClassName = defaultNginxStorageClassName
 			scParameters[svStorageClassName] = storagePolicyName
-			svcClient, svNamespace := getSvcClientAndNamespace()
```

Comment:
i case of GC you need this step. Why this is removed?

Reply:
this is already part of BeforeEach() call, hence removed from here